docs/docs/guides/developer-guide/channel-aware/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Channelaware entities"
```

Comment:
To be consistent I think you should also hyphenate `channel-aware` here

Reply:
I had it before like that and it kinda looked "ugly" in the sidebar navigation to be honest.

Reply:
![image](https://github.com/user-attachments/assets/9fdbd481-1d0d-4c92-a7e1-82f0d6dfc7bb)
@michaelbromley How about "Implementing ChannelAware" and then we can have the same titles for Translateable and HasCustomFields?

Or is this maybe even worth a new "sub menu"?

Reply:
yes good idea